kfp/transform_workflows/Makefile

```diff
@@ -1,59 +1,44 @@
 # Define the root of the local git clone for the common rules to be able
 # know where they are running from.
-REPOROOT=../..
+export REPOROOT := ${CURDIR}/../..
 include ${REPOROOT}/.make.versions
 
 CURR_DIR := ${CURDIR}
 
 IGNORE := $(shell bash -c "sed -n /=/p  ${REPOROOT}/kfp/requirements.env | sed 's/=/:=/' | sed 's/^/export /' > makeenv")
 
-
 include makeenv
 # Include the common rules.
```

Comment:
Where is makeenv?

Reply:
This part is removed (with additional Makefile cleanup) Thanks